apps/frontend/src/libs/components/table/table.tsx

```diff
@@ -129,7 +130,10 @@ const Table = <T extends object>({
 								)}
 								{row.getVisibleCells().map((cell) => (
 									<td
-										className={styles["table-data"]}
+										className={getValidClassNames(
+											styles["table-data"],
+											styles[getCellColor(cell.getValue())],
```

Comment:
I think that we might as well omit creation of this helper. `GetValidClassNames` provides us conditional classNames adding. See how it's done in other places, for example our `Select` component.

Reply:
We shouldn't have this logic in table component at all, if we need to change color, we need to wrap the item with the span and apply the color to it and pass as a cell to columns config.